logging.adoc

```diff
@@ -0,0 +1,397 @@
+kubernetes.pod_name,	    resource.attributes.k8s.pod.name
+log_source,	                resource.attributes.openshift.log.source
+kubernetes.container_name,	resource.attributes.k8s.container.name
+--,--
+@timestamp,	                logRecords.timeUnixNano
+N/A,	                    logRecords.observedTimeUnixNano
+level,	                    logRecords.severityNumber
+message,                    logRecords.body
+--,--
+log_type,	                logRecords.attributes.openshift.log.type
+url,                        logRecords.attributes.url.full
+response.code,              logRecords.attributes.http.response.status.code
+verb,                       logRecords.attributes.http.request.method
+kubernetes.pod_id,	        logRecords.attributes.k8s.pod.uid
+kubernetes.container_id,	logRecords.attributes.k8s.container.id
+--,--
+systemd.t.PID,                    logRecords.attributes.syslog.procid
+systemd.u.SYSLOG.FACILITY,        logRecords.attributes.syslog.facility
+systemd.u.SYSLOG.IDENTIFIER,      logRecords.attributes.syslog.identifier
+systemd.t.SYSTEMD.CGROUP,         logRecords.attributes.system.cgroup
+systemd.t.SYSTEMD.INVOCATION.ID,  logRecords.attributes.system.invocation.id
+systemd.t.SYSTEMD.SLICE,          logRecords.attributes.system.slice
+systemd.t.SYSTEMD.UNIT,           logRecords.attributes.system.unit
+--,--
+systemd.t.BOOT_ID,                logRecords.attributes.system.boot.id
+systemd.t.COMM,                   logRecords.attributes.system.comm
+systemd.t.CAP_EFFECTIVE,          logRecords.attributes.system.cap.effective
+systemd.t.CMDLINE,                logRecords.attributes.system.cmdline
+systemd.t.EXE,                    logRecords.attributes.system.exe
+systemd.t.GID,                    logRecords.attributes.system.gid
+systemd.t.MACHINE_ID,             logRecords.attributes.system.machine.id
+systemd.t.SELINUX_CONTEXT,        logRecords.attributes.system.selinux.content
+systemd.t.STREAM_ID,              logRecords.attributes.system.stream.id
+systemd.t.TRANSPORT,              logRecords.attributes.system.transport
+|===
+
+Please note:  More work to refine this list is necessary and it will likely change a few times.  Please do not accept this as a specified data model.
+
+* https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md[General Attributes]
+
+.Viaq Data Model
+* https://github.com/openshift/cluster-logging-operator/blob/master/docs/reference/datamodels/viaq/v1.adoc
+
+
+---
+
+
+== A Working Example
+Writing to a OTEL Collector running in another namespace.
+
+.Technical Preview
+An annotation must be added the forwarder definition
+`_observability.openshift.io/tech-preview-otlp-output: "enabled"_`
+
+
+== Steps
+Create a configuration, service and a pod running an image of the otel-collector and forward logs
+
+
+. Create a ConfigMap instance containing a `config.yaml` for the otel-collector.
++
+----
+ oc apply -f otel-receiver-config.yaml
```

Comment:
it would make more sense to use OpenTelemetryCollector CRD which is a supported RH product.

Reply:
I can do this yes.   Ironically, I had no idea we supported anything like that.  It would have helped.